grpc/federation/cel/list.go

```diff
@@ -1,15 +1,26 @@
 package cel
 
 import (
+	"sort"
+
 	"github.com/google/cel-go/cel"
 	"github.com/google/cel-go/common/ast"
 	"github.com/google/cel-go/common/operators"
 	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/google/cel-go/common/types/traits"
 	"github.com/google/cel-go/parser"
 )
 
 const ListPackageName = "list"
 
+const (
+	listSortAscFunc        = "grpc.federaiton.list.@sortAsc"
```

Comment:
📝  I intend the function with `@` prefix as a private. I referred it from [ext/math.go](https://github.com/google/cel-go/blob/master/ext/math.go#L338), but this is just a mark and does not appear to be a restriction.